uv.lock

```diff
@@ -1891,6 +1891,15 @@ wheels = [
     { url = "https://files.pythonhosted.org/packages/21/2c/5e05f58658cf49b6667762cca03d6e7d85cededde2caf2ab37b81f80e574/pillow-11.2.1-pp311-pypy311_pp73-win_amd64.whl", hash = "sha256:208653868d5c9ecc2b327f9b9ef34e0e42a4cdd172c2988fd81d62d2bc9bc044", size = 2674751, upload_time = "2025-04-12T17:49:59.628Z" },
 ]
 
+[[package]]
+name = "pkgconfig"
```

Comment:
This is a bit of an odd one. It is marked as a build dependency for `pyvips` packaged. I updated my local `uv` and the Python version to see if it got excluded. However, I suspect the inclusion might be based off MacOS vs Linux (Dependency update GitHub action). A few additional local-package upgrades later I've thrown in the towel for such a small abbaration.